### ISSUE TITLE
Fails when ios project was never built (on Windows)

### ISSUE THREAD

**Issue by pke** [unimonkiez/react-native-asset#29]
This is related to https://github.com/react-native-community/cli/issues/1173

```bash
$ npx react-native-asset
info Linking otf assets to iOS project 
WARN ERRGROUP Group 'Resources' does not exist in your Xcode project. We have created it automatically for you.
Cannot read property 'UIAppFonts' of null
```

Maybe this tool could print out a better error message?

**Comment by unimonkiez** [comment]
It seems like something with the linking, [try this](https://github.com/facebook/react-native/issues/21432)

**Comment by pke** [comment]
> It seems like something with the linking, [try this](https://github.com/facebook/react-native/issues/21432)

No, this does not solve it. The problem is described in https://github.com/react-native-community/cli/issues/1173

I do NOT HAVE an ios project in my app.

**Comment by unimonkiez** [comment]
I haven't noticed the title the first time I read the issue.  
So we need to add the ability to ignore android / ios projects when they are not present.  
Seems like an easy PR, if you want to do it.
